src/icons/homescreenLogo.js

```diff
@@ -1,28 +1,36 @@
+
+const StyledLogodiv = styled.div`
+  justify-content: center;
+  display: flex;
+`;
 
 export default function HomescreenLogo() {
   return (
-    <svg width="91.196" height="105.017" viewBox="0 0 91.196 105.017">
-      <defs>
-        <linearGradient
-          id="linear-gradient"
-          x1="0.485"
-          y1="0.999"
-          x2="0.541"
-          y2="0.261"
-          gradientUnits="objectBoundingBox"
-        >
-          <stop offset="0" stop-color="#00f5ff" />
-          <stop offset="1" stop-color="#800040" />
-        </linearGradient>
-      </defs>
-      <path
-        id="Subtraction_4"
-        data-name="Subtraction 4"
-        d="M99.753,152.017h0L84.7,143.623l9.835-18.638,18.335,19.1-13.114,7.935Zm15.021-9.089L95.6,122.96l10.224-19.374,9.549-18.1,22.239,30.788-8.267,17.832-14.572,8.818h0ZM82.9,142.622h0l-10.912-6.084L81.2,111.1l11.838,12.328L82.9,142.621Zm-12.664-7.061h0l-16.107-8.98-.747-44.455L79.668,109.5l-9.43,26.058Zm62.121-3.274v0l6.573-14.178,5.129,7.1-11.7,7.08Zm12.085-10.253,0,0-4.464-6.18,4.209-9.079.256,15.258ZM94.1,121.4h0l-12.08-12.582,16.755-46.3,15.255,21.119L94.1,121.4Zm44.556-7.373h0L116.5,83.354,127.116,63.24l16.495,9.2.5,29.825-5.453,11.762Zm-58.165-6.8h0L53.33,78.932l-.082-4.859L90.74,51.387l6.635,9.185-16.881,46.65ZM115.16,81.494h0L99.669,60.049l3.647-10.078,22.005,12.268L115.16,81.493ZM98.266,58.108v0L92.6,50.262,97.988,47l3.576,1.994-3.3,9.111Z"
-        transform="translate(-53.249 -47.001)"
-        fill="url(#linear-gradient)"
-      />
-    </svg>
+    <StyledLogodiv>
+      <svg width="91.196" height="105.017" viewBox="0 0 91.196 105.017">
+        <defs>
+          <linearGradient
+            id="linear-gradient"
+            x1="0.485"
+            y1="0.999"
+            x2="0.541"
+            y2="0.261"
+            gradientUnits="objectBoundingBox"
+          >
+            <stop offset="0" stop-color="#00f5ff" />
+            <stop offset="1" stop-color="#800040" />
+          </linearGradient>
+        </defs>
+        <path
+          id="Subtraction_4"
+          data-name="Subtraction 4"
+          d="M99.753,152.017h0L84.7,143.623l9.835-18.638,18.335,19.1-13.114,7.935Zm15.021-9.089L95.6,122.96l10.224-19.374,9.549-18.1,22.239,30.788-8.267,17.832-14.572,8.818h0ZM82.9,142.622h0l-10.912-6.084L81.2,111.1l11.838,12.328L82.9,142.621Zm-12.664-7.061h0l-16.107-8.98-.747-44.455L79.668,109.5l-9.43,26.058Zm62.121-3.274v0l6.573-14.178,5.129,7.1-11.7,7.08Zm12.085-10.253,0,0-4.464-6.18,4.209-9.079.256,15.258ZM94.1,121.4h0l-12.08-12.582,16.755-46.3,15.255,21.119L94.1,121.4Zm44.556-7.373h0L116.5,83.354,127.116,63.24l16.495,9.2.5,29.825-5.453,11.762Zm-58.165-6.8h0L53.33,78.932l-.082-4.859L90.74,51.387l6.635,9.185-16.881,46.65ZM115.16,81.494h0L99.669,60.049l3.647-10.078,22.005,12.268L115.16,81.493ZM98.266,58.108v0L92.6,50.262,97.988,47l3.576,1.994-3.3,9.111Z"
+          transform="translate(-53.249 -47.001)"
+          fill="url(#linear-gradient)"
+        />
+      </svg>
```

Comment:
I think you should consider importing the `svg`  instead of having all this code in here

Reply:
good idea, i will do that, thx. the code in general needs a bit of a clean up anyway